entity-framework/core/modeling/relationships.md

```diff
@@ -1,338 +1,196 @@
-
-[!code-csharp[Main](../../../samples/core/Modeling/Relationships/OneToOne.cs?name=OneToOne&highlight=6,15-16)]
-
-> [!NOTE]
-> EF will choose one of the entities to be the dependent based on its ability to detect a foreign key property. If the wrong entity is chosen as the dependent, you can use the Fluent API to correct this.
-
-When configuring the relationship with the Fluent API, you use the `HasOne` and `WithOne` methods.
-
-When configuring the foreign key you need to specify the dependent entity type - notice the generic parameter provided to `HasForeignKey` in the listing below. In a one-to-many relationship it is clear that the entity with the reference navigation is the dependent and the one with the collection is the principal. But this is not so in a one-to-one relationship - hence the need to explicitly define it.
+    public ICollection<Post> Posts { get; }
+}
+```
 
-[!code-csharp[Main](../../../samples/core/Modeling/Relationships/FluentAPI/OneToOne.cs?name=OneToOne&highlight=11)]
+```csharp
+public class Post
+{
+    public int Id { get; set; }
+    public string Title { get; set; }
+    public string Content { get; set; }
+    public DateTime PublishedOn { get; set; }
+    public bool Archived { get; set; }
+
+    public int BlogId { get; set; }
+    public Blog Blog { get; set; }
+}
+```
 
-The dependent side is considered optional by default, but can be configured as required. However EF will not validate whether a dependent entity was provided, so this configuration will only make a difference when the database mapping allows it to be enforced. A common scenario for this are reference owned types that use table splitting by default.
+> [!TIP]
+> Primary and foreign key properties don't need to publicly visible properties of the entity type. However, even when the properties are hidden, it is important to recognize that they still exist in the EF model.
+
+The primary key property of `Blog`, `Blog.Id`, and the foreign key property of `Post`, `Post.BlogId`, can then be associated with the references ("navigations") between the entity types (`Blog.Posts` and `Post.Blog`). This is done automatically by EF when building a simple relationship like this, but can also be specified explicitly when overriding the `OnModelCreating` method of your `DbContext`. For example:
+
+```csharp
+protected override void OnModelCreating(ModelBuilder modelBuilder)
+{
+    modelBuilder.Entity<Blog>()
+        .HasMany(e => e.Posts)
+        .WithOne(e => e.Blog)
+        .HasForeignKey(e => e.BlogId)
+        .HasPrincipalKey(e => e.Id);
+}
+```
 
-[!code-csharp[Main](../../../samples/core/Modeling/OwnedEntities/OwnedEntityContext.cs?name=Required&highlight=12-13)]
+Now all these properties will behave coherently together as a representation of a single relationship between `Blog` and `Post`.
 
-With this configuration the columns corresponding to `ShippingAddress` will be marked as non-nullable in the database.
+## Find out more
```

Comment:
Don't we have a standard title for this? Like "See more" or "Additional reading"?

Reply:
Maybe, although I think "Additional resources" is somewhat different. This is very directed next steps. Happy to call it whatever.